plugins/admin/README.md

```diff
@@ -1,11 +1,8 @@
-# admin
+# @leancodepl/contractsgenerator-typescript-plugin-admin
```

Comment:
IMO every plugin should also have documentation on its own. At least params that it takes in.